app/components/provider_interface/application_card_component.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="app-application-card">
+
+  <div class="app-application-card__primary">
+    <h3 id="applicant-name-<%= index %>" class="govuk-heading-m govuk-!-margin-bottom-3">
+      <%= link_to candidate_name, provider_interface_application_choice_path(application_choice), class: 'govuk-link' %>
+    </h3>
+    <p id="course-provider-name-<%= index %>" class="govuk-body govuk-!-font-size-16"><%= course_provider_name %></p>
+    <p id="course-name-and-code-<%= index %>" class="govuk-body govuk-!-margin-bottom-0"><%= course_name_and_code %></p>
+    <p id="accredited-body-<%= index %>" class="govuk-body govuk-caption-m govuk-!-font-size-16">Accredited body: <%= accrediting_provider.name if accrediting_provider %></p>
+  </div>
+
+  <div class="app-application-card__secondary">
+    <div id="status-<%= index %>" class="app-application-card__status">
+      <%= render ProviderInterface::ApplicationStatusTagComponent.new(application_choice: application_choice) %>
+    </div>
+    <div>
+      <p id="updated-at-<%= index %>" class="govuk-!-margin-bottom-0 govuk-caption-m govuk-!-font-size-16">Updated on <%= updated_at %></p>
```

Comment:
There are too many IDs in this component; for testing purposes, we could just have one `id` at the top level and then refer to the nested elements using other CSS selectors.

If it's solely used for testing, I also prefer using `data-qa="application-card-13"` as it communicates the intention better.

Reply:
Okay cool, will refactor.

Reply:
👍